docs/chapters/03.txt

```diff
@@ -475,16 +472,26 @@ IEEE Binary Floats
 ==================
 
 According to the IEEE Standard for Binary Floating-Point Arithmetic, "single-"
-and "double-precision" numbers are defined as follows:
+and "double-precision" numbers are defined as follows.
 
-::
++-------------+----------+------------+-------------+----------------+-------------+
+| ::          | ::       | ::         | ::          | ::             | ::          |
```

Comment:
all of the cells in this header row should be marked up like so:

```
| ``EXAMPLE``   |
```

Reply:
If I do it like that ``EXAMPLE``, than it will look the same on the internet